### PR TITLE
Fix circular imports created by alphabetization

### PR DESCRIPTION
**Pull request by ryanjulian**
This PR also adds an automated import test to prevent future breakages
from circular imports.

### CODE REVIEW

File: scripts/travisci/check_imports.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import ast
+import glob
+import logging
+import os
```
Comment (gonzaiva) [inline scripts/travisci/check_imports.py]: Not used

File: scripts/travisci/check_imports.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        except SyntaxError as e:
+            logging.error(("{filename}:{line_num} - \"{line}\" "
+                           "has a SyntaxError.").format(
+                               filename=fn,
+                               line_num=e.lineno,
+                               line=lines[node.lineno - 1].strip()))
+            passed = False
+
+        for node in ast.walk(module):
+            if isinstance(node, ast.Import) or isinstance(
```
Comment (gonzaiva) [inline scripts/travisci/check_imports.py]: isinstance(node, (ast.Import, ast.ImportFrom))

File: rllab/sampler/utils.py [hunk 3]
```diff
@@ -403,7 +403,7 @@ def max_size_tests():
         print("passed")
 
 
-def test_memory_usage_ok():
+def test_memory_usage_ok(q):
```
[the remark this answers is not in the document]
Comment (ryanjulian) [reply rllab/sampler/utils.py]: Typo. Good catch.

File: scripts/travisci/check_imports.py (hunk 1 above)
Comment (ryanjulian) [reply scripts/travisci/check_imports.py]: done

File: scripts/travisci/check_imports.py (hunk 2 above)
Comment (ryanjulian) [reply scripts/travisci/check_imports.py]: done.

File: rllab/sampler/utils.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ryanjulian) [reply rllab/sampler/utils.py]: done

File: rllab/envs/mujoco/maze/maze_env_utils.py [hunk 4]
```diff
@@ -7,8 +7,6 @@
 
 from rllab.misc import logger
 
-mpl.use('Agg')
```
[the remark this answers is not in the document]
Comment (ryanjulian) [reply rllab/envs/mujoco/maze/maze_env_utils.py]: done.

**Pull request merged**